src/document/DocumentCommandHandlers.js

```diff
@@ -1205,7 +1205,12 @@ define(function (require, exports, module) {
         if (doc && doc.isDirty && !_forceClose && (MainViewManager.isExclusiveToPane(doc.file, paneId) || _spawnedRequest)) {
             // Document is dirty: prompt to save changes before closing if only the document is exclusively
             // listed in the requested pane or this is part of a list close request
-            var filename = FileUtils.getBaseName(doc.file.fullPath);
+            var filename = FileUtils.getBaseName(doc.file.fullPath),
+                disableSaveBtn = "";
+
+            if (file.toString().indexOf("[RemoteFile") === 0) {
```

Comment:
We should use a better way for RemoteFile instance detection.

Reply:
I tried instanceof that's not working for me. Is there anything which we can use?